src/components/include/resumption/last_state_wrapper.h

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2018, Ford Motor Company
```

Comment:
@YarikMamykin 2019

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/04c5cfc370f27b391e68d65c0f15c5f6fdace435